### DISCUSSION TITLE
Unicorn direct view

### DISCUSSION THREAD

**Discussion by Fidel-C** [django-commons/django-unicorn#322] [General]
I really like the new feature.

But how how do you combine it with the root template (index.html) and probably extend a base.html file.
How to use UnicornView.as_view()

**Comment by adamghill** [comment]
Thanks! You should be able to extend and use blocks as you normally would with Django templates.

Template example: https://github.com/adamghill/django-unicorn/blob/master/example/unicorn/templates/unicorn/direct-view.html
urls.py example: https://github.com/adamghill/django-unicorn/blob/master/example/www/urls.py#L12-L16
